src/components/generic/OptionsList/OptionsList.jsx

```diff
@@ -27,6 +28,15 @@ export default function OptionsList({
   withKey,
 }) {
   const [cssClasses, sc] = webexComponentClasses('options-list', className);
+  const optionsListRef = useRef();
+  const elementPosition = useElementPosition(optionsListRef);
+  const [maxHeight, setMaxHeight] = useState(0);
+
+  useEffect(() => {
+    if (elementPosition.top) {
```

Comment:
top=0 might be a valid value in some instances, maybe you can just return `null` from the hook if target not yet exists (and as the initial value) instead of an empty object